bindings/protocols/coap/index.html

```diff
@@ -338,21 +361,41 @@ <h3>Observing Resources</h3>
                 "cov:method": "GET",
                 "href": "coap://[2001:DB8::1]/status",
                 "contentType": "text/plain;charset=utf-8",
-                "subprotocol": "cov:observe",
-                "op": ["observeproperty"]
+                "op": ["observeproperty", "unobserveproperty"]
             }]
         }
     }
 }
 </pre>
             <p>
-                A consumer can always fall back to polling a resource if observing a resource is permanently or temporarily not
+                For the `observeproperty` and `subscribeevent` operations, a consumer SHOULD add an Observe option to GET or FETCH requests by default.
+                Note, however, that using FETCH for the `observeproperty` operation is NOT RECOMMENDED as FETCH requests must contain a Content-Format option which can imply that the request payload must not be empty.
+                For the `subscribeevent` operation on the other hand, using FETCH is RECOMMENDED if the Event Affordance expects the consumer to provide data described by the `subscription` field.
+            </p>
+            <p>
+                If the server's response does not contain an Observe option this indicates that the requested server does not support observe and that the consumer SHOULD fall back to polling instead (see section [[[#polling-resources]]]).
```

Comment:
```suggestion
                For the `observeproperty` and `subscribeevent` operations, a consumer SHOULD add an `Observe` option to `GET` or `FETCH` requests.
                Note, however, that using `FETCH` for the `observeproperty` operation is NOT RECOMMENDED, as `FETCH` requests must contain a `Content-Format` option which can imply that the request payload must not be empty.
                For the `subscribeevent` operation, on the other hand, using `FETCH` is RECOMMENDED if the Event Affordance expects the consumer to provide data described by the `subscription` field.
            </p>
            <p>
                If the server's response does not contain an `Observe` option, this indicates that the requested server does not support observe, and that the consumer SHOULD fall back to polling instead (see section [[[#polling-resources]]]).
```

Reply:
The remaining comments on the `index.html` file should be resolved now as well by regenerating the rendered document :) Thank you once more for the very helpful suggestions!